iOS/RemoteCollectionController.swift

```diff
@@ -4,7 +4,8 @@ class RemoteCollectionController: UICollectionViewController {
     var sections = Photo.constructRemoteElements()
 
     lazy var overlayView: UIView = {
-        let view = UIView()
+        // Needs a frame, otherwise it won't show up.
```

Comment:
It gets a frame here, so it shouldn't be needed but UIKit = :dolan:

https://github.com/bakkenbaeck/SectionScrubber/blob/1a48e4e39e5053a778254292211e3f8bfe3afc70/iOS/RemoteCollectionController.swift#L74